cmd/main.go

```diff
@@ -108,10 +108,10 @@ func main() {
 	}
 
 	listen := ":3333"
-	globalLogger.Info("Starting QuickPizza", zap.String("listenAddress", listen))
+	slog.Info("Starting QuickPizza", "listenAddress", listen)
 	err = http.ListenAndServe(listen, server)
 	if err != nil {
-		globalLogger.Error("Running HTTP server", zap.Error(err))
+		slog.Error("Running HTTP server", "err", err)
```

Comment:
We probably want `os.Exit` here as well. The original line was not `Fatal` but I think it should always have been 🌍🧑‍🚀🔫 :)